### PR TITLE
Added khmer language

### PR DESCRIPTION
**Pull request by HELMAB**

### CODE REVIEW

**Comment by rappasoft** [comment]
Bump

File: resources/lang/km/passwords.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    | that are given by the password broker for a password update attempt
+    | has failed, such as for an invalid token or invalid new password.
+    |
+    */
+
+    'reset' => 'ពាក្យសម្ងាត់បានផ្លាស់ប្តូរជាថ្មី។',
+    'sent' => 'ពាក្យសម្ងាត់ដែលបានរំលឹកបានផ្ញើរួចរាល់!',
+    'throttled' => 'សូមរង់ចាំមុនពេលសាកល្បងម្តងទៀត។',
+    'token' => 'ពាក្យសម្ងាត់ដែលកំណត់ឡើងវិញគឺមិនត្រឹមត្រូវទេ។',
+    'user' => "យើងមិនអាចរកឃើញអ្នកប្រើដែលមានអាសយដ្ឋានអ៊ីមែលមួយនេះទេ។",
```
Comment (coderkoala) [inline resources/lang/km/passwords.php:20]: @HELMAB you might wanna look into this particular line, as StyleCI is flagging this as the check fail.

**Review by coderkoala — CHANGES_REQUESTED** [review]
You might wanna see if you can fix the StyleCI issues before this can even be looked at by Anthony.

File: resources/lang/km/passwords.php (hunk 1 above)
Comment (HELMAB) [reply resources/lang/km/passwords.php]: Thank you @coderkoala I have updated it

**Pull request merged**